cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlan.cpp

```diff
@@ -175,4 +177,29 @@ bool VeloxPlanFragmentToSubstraitPlan::shouldAppendValuesNode(
          !std::dynamic_pointer_cast<const core::AbstractJoinNode>(sourceNode);
 }
 
+VeloxPlanFragmentToSubstraitPlan::VeloxPlanFragmentToSubstraitPlan() {
+  if (FLAGS_substrait_yaml_root_path != "") {
+    static const std::vector<std::string> extensionFiles = {
```

Comment:
Do we need to match exactly the file names? Or just iterate over the files under ```FLAGS_substrait_yaml_root_path``` without touching files.

Reply:
Once we merge W43 into main branch ,  these code will removed , currently just workaround

Reply:
sounds good.